training/4-train-lstm.py

```diff
@@ -28,12 +32,12 @@
 # +
 def load_filenames():
     filenames = list()
-    with h5py.File(os.joind(PREDICTION_ROOT, 'prediction_test_iclabel.h5'), 'r') as f:
+    with h5py.File(os.path.join(PREDICTION_ROOT, 'prediction_test_iclabel.h5'), 'r') as f:
         filenames = list(f['filenames'])
     return filenames
 
 
-def prepare_file(file_i, filename, classifiers, f_nick, model_type):    
+def prepare_file(file_i, filename, classifiers, f_nick, model_type, fs):    
```

Comment:
sure I am guilty of the same but I don't like functions without description a docstring (do you what you want of comment)

Reply:
Mostly fixed with 70ae9d6b5ae88bdf72495cb94c61bd085a904961